src/main/kotlin/string/StringExplosion.kt

```diff
@@ -0,0 +1,36 @@
+package string
+
+import java.io.BufferedWriter
+import java.io.OutputStreamWriter
+import java.util.Stack
+
+fun main() {
+    var str = readln()
+    val bomb = readln()
+    val stack = Stack<Char>()
+    val answer = StringBuilder()
+    val bw = BufferedWriter(OutputStreamWriter(System.out))
+
+    str.forEach {
+        stack.push(it)
+        if (stack.size >= bomb.length && stack.peek() == bomb.last()) {
+            var bool = true
+            bomb.forEachIndexed { index, c ->
+                if (stack[stack.size - bomb.length + index] != c) {
+                    bool = false
+                    return@forEachIndexed
+                }
+            }
+            if (bool) {
+                repeat(bomb.length) {
+                    stack.pop()
+                }
+            }
+        }
+    }
+    stack.forEach {
+        answer.append(it)
+    }
+    if (answer.isNotEmpty()) bw.write(answer.toString()) else bw.write("FRULA")
```

Comment:
```suggestion
    if (stack.isNotEmpty()) bw.write(stack.joinToString("")) else bw.write("FRULA")
```

answer 변수를 사용하지 않고 stack 변수를 사용해서 위와 같이 수정 가능해보입니다 :)